### PR TITLE
KIALI-2850 Disable displaying tracing information in the UI

### PR DESCRIPTION
**Pull request by aljesusg**
JIRA: [KIALI-2850]( https://issues.jboss.org/browse/KIALI-2850)

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -41,6 +41,7 @@ const (
 	EnvServerMetricsPort                = "SERVER_METRICS_PORT"
 	EnvServerMetricsEnabled             = "SERVER_METRICS_ENABLED"
 
+	EnvGrafanaEnabled                  = "GRAFANA_ENABLED"
```
Comment (jotak) [inline config/config.go]: The `EnvGrafanaDisplayLink` already does pretty much the same, so I think either don't add the new one or remove the old one.

Comment (aljesusg) [reply config/config.go]: Removed. Thanks @jotak

**Comment by aljesusg** [comment]
Now the information is removed in /api/status when is disabled

File: operator/deploy/kiali/kiali_cr.yaml [hunk 2]
```diff
@@ -191,12 +191,14 @@ spec:
 #      url: ""
 # **Tracing-specific settings:
 #  - Right now we only support Jaeger
+# enabled: When true, Kiali shows Jaeger and autodiscover
```
Comment (jmazzitelli) [inline operator/deploy/kiali/kiali_cr.yaml]: Trivial - but can we reword this slightly? The sentence seems to be cut off here. Say something like:

```suggestion
# enabled: When true, Kiali shows Jaeger and will attempt to autodiscover it.
```

File: operator/deploy/kiali/kiali_cr.yaml [hunk 3]
```diff
@@ -191,12 +191,14 @@ spec:
...
 # namespace: The Kubernetes namespace that holds the Tracing service (if empty, assumes the same value as deployment.namespace)
 # service: The Kubernetes service name for tracing. Kiali uses this to connect within the cluster to Jaeger.
 # url: The URL that Kiali uses when integrating with Tracing. This URL must be accessible to clients external to
 #      the cluster in order for the integration to work properly. If empty, an attempt to auto-discover it is made.
 #    ---
 #    tracing:
+#      enabled: "true"
```
Comment (jmazzitelli) [inline operator/deploy/kiali/kiali_cr.yaml]: looking at the defaults/main.yml, the default is actually `true` not `"true"` (actual boolean, not string). So this should be:

```suggestion
#      enabled: true
```

Comment (aljesusg) [reply operator/deploy/kiali/kiali_cr.yaml]: so display_link should be boolean too. I found it strange to see it

File: operator/roles/kiali-deploy/defaults/main.yml [hunk 4]
```diff
@@ -58,6 +58,7 @@ kiali_defaults:
       custom_metrics_url: ""
       url: ""
     tracing:
+      enabled: true
```
Comment (jmazzitelli) [inline operator/roles/kiali-deploy/defaults/main.yml]: Just a question: do you really want `true` here and not the string `"true"`? - see my comment above

**Comment by aljesusg** [comment]
done @jmazzitelli

**Pull request merged**